xnmt/embedder.py

```diff
@@ -55,7 +57,8 @@ def __init__(self, yaml_context, vocab_size, emb_dim = None, weight_noise = None
     self.weight_noise = weight_noise or yaml_context.weight_noise
     self.word_dropout = word_dropout
     self.fix_norm = fix_norm
-    self.embeddings = yaml_context.dynet_param_collection.param_col.add_lookup_parameters((self.vocab_size, self.emb_dim))
+    self.embeddings = yaml_context.dynet_param_collection.param_col.add_lookup_parameters((self.vocab_size, self.emb_dim),
+                                                                                          init=linear_init(self.vocab_size))
```

Comment:
I did several runs with and without this LeCun initialisation and found that with this initialisation, model's convergence improved in transformer model.

Reply:
Sure, that's good. But this is in `embedder.py` so it will influence all models in xnmt. I think perhaps we should have this as an option. Then we can change the default when we're sure it's good in all models.